web/src/features/panels/zone/ZoneHeaderTitle.tsx

```diff
@@ -44,19 +44,24 @@ export default function ZoneHeaderTitle({
                 size={18}
                 className="shadow-[0_0px_3px_rgba(0,0,0,0.2)]"
               />
-              <div className="ml-2 flex flex-row">
-                <h2
-                  className="max-w-[300px] overflow-hidden truncate text-lg font-medium sm:max-w-[230px] md:max-w-[270px]"
-                  data-test-id="zone-name"
-                >
-                  {title}
-                </h2>
-                {isSubZone && (
-                  <p className="ml-2 flex w-auto items-center whitespace-nowrap rounded-full bg-gray-200 py-0.5 px-2  text-sm dark:bg-gray-900">
-                    {countryName || zoneId}
-                  </p>
-                )}
-              </div>
+              <TooltipWrapper
+                tooltipContent={title.length > 20 ? title : undefined}
+                side="bottom"
```

Comment:
Looks good but why are we restricting this to just zones with names longer than 20 characters?

I think it would make more sense to apply it to all zone names regardless of length to make the experience more consistent across the zones, so the users always get what they expect.

I may or may not just have spent 3 weeks working with this subject so I might be slightly biased. 😅

Reply:
Zone names above 20 characters are truncated.

Maybe it's just me but a tooltip that says exactly what the trigger says seems a little silly 😆 

<img width="168" alt="image" src="https://user-images.githubusercontent.com/18622768/225133176-299c71e6-8d32-4304-a073-59fa42ec8b1f.png">

Reply:
Then we have another problem to take into account as well. 👀
![image](https://user-images.githubusercontent.com/30777521/225133833-2308d308-fd41-43c3-b0db-9e9dadda353d.png)
Here it is shown but the zone name is not truncated.

Reply:
Ah okay, so it's not always truncated if they are above 20 characters (if there is available screen space) but the tooltip is always shown if the zone name is above 20 characters. 

(Just summarizing things so far.) 

In that case I think we should go either with:

- only showing the tooltip if the zone name is truncated

or:

- Always show the tooltip regardless of zone name length.

Both have benefits and downsides so I'm not sure which is the best one going forward but the second option is easier to implement.